FirebaseMessaging.podspec

```diff
@@ -36,10 +36,14 @@ device, and it is completely free.
     'Firebase/InstanceID/Public/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
   ]
-  s.requires_arc = [
-    base_dir + 'Sources/*.m',
-    base_dir + 'Sources/Token/*.m'
-  ]
+
+  # TODO: Find out why `requires_arc` was required. 
```

Comment:
I suspect, it's left over from the when the Protobuf dependency required some files to be built without ARC

Reply:
If so, looks like safe to be removed.

Reply:
Agreed